src/Accord.Bot/Infrastructure/ReminderService.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using MediatR;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Remora.Discord.API.Abstractions.Rest;
+using Remora.Discord.Core;
+
+namespace Accord.Services.Reminder
+{
+    public class ReminderService : BackgroundService
```

Comment:
Could you rename this to `RemindersHostedService` just so that it's not confused as a general service for database stuff?

Reply:
sure done